source/install/native.rst

```diff
@@ -71,6 +72,12 @@ like for the environment. In this example, we'll name the environment
    conda env create -n qiime2-2022.11 --file qiime2-2022.11-py38-osx-conda.yml</pre>
    OPTIONAL CLEANUP
    <pre>rm qiime2-2022.11-py38-osx-conda.yml</pre>
+         </div>
+         <div id="M1-macOS" class="tab-pane fade">
+            <pre>wget https://data.qiime2.org/distro/core/qiime2-2022.11-py38-osx-conda.yml
+   CONDA_SUBDIR=osx-64 conda env create -n qiime2-2022.11 --file qiime2-2022.11-py38-osx-conda.yml
+   conda activate qiime2-2022.11
+   conda config --env --set subdir osx-64</pre>
```

Comment:
Should we have the usual optional cleanup block after this?

Reply:
oh whoops, yes that's a good idea. adding now!